src/CarbonAware.Aggregators/test/CarbonAwareParametersTests.cs

```diff
@@ -219,7 +217,7 @@ public void PropertyNameEnum_MatchesModelAndDtoClassProperties()
         // Arrange
         var dtoClassPropertyNames = typeof(CarbonAwareParametersBaseDTO).GetProperties().Select(p => p.Name);
         var modelClassPropertyNames = typeof(CarbonAwareParameters).GetProperties().Select(p => p.Name);
-        var enumPropertyNames = Enum.GetNames<PropertyName>();
+        var enumPropertyNames = CarbonAwareParameters.GetPropertyNames();
```

Comment:
Does this test pass this way even though `enumPropertyNames` is an `IEnumerable<PropertyName>` type and the other two vars are `IEnumerable<string>`types?